routes/api/profile.js

```diff
@@ -1,11 +1,61 @@
 const express = require('express');
 const router = express.Router();
+const mysql = require('mysql');
+const bodyParser = require('body-parser');
+var user = require('./user')
 
 //@route GET api/profile/test
 //@desc Test profile route
 //@access Public
+router.use(bodyParser.json());
+router.use(bodyParser.urlencoded({ extended: true }));
+
 router.get('/test', (req, res) => res.json({msg: 'Profile works'}));
 
+function getMySQLConnection() {
+    return mysql.createConnection({
+        host     : 'localhost',
+        user     : 'root',
+        password : '',
+        port     : '3307',
+        database : 'ETES'
+    });
+}
+
+
 
+router.post('/', function(req, res){
```

Comment:
Hi Colman,
Is there any reason why this method is post instead of get? And besides purchase history, the user may sell tickets, so could you please return selling history also? It is ok for selling history to be empty.